src/ducks/notifications/services.spec.js

```diff
@@ -0,0 +1,36 @@
+jest.mock('handlebars')
```

Comment:
It was forgotten

Reply:
It's here on purpose because handlebars was throwing an error in test context. And since we are not testing the HTML part here, I just mocked it for now. We can see what's wrong with handlebars in tests when we try to test the HTML rendering.

Reply:
maybe you can add a comment?

Reply:
Yep, here it is !